project.clj

```diff
@@ -6,17 +6,19 @@
   :dependencies [[org.clojure/clojure "1.5.1"]
                  [crypto-random "1.2.0"]
                  [crypto-equality "1.0.0"]
-                 [commons-codec "1.10"]
-                 [org.mindrot/jbcrypt "0.3m"]
+                 [commons-codec "1.12"]
```

Comment:
Why has this been updated?

Reply:
Just to be up to date instead of using the 5year old version.
I now removed it from this PR to keep the changes separated.

Reply:
It's a good idea, but PRs should be focused on a single change.